x-pack/plugins/cases/public/components/connectors/thehive/types.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum TheHiveTLP {
```

Comment:
I think is better if we do

```
export enum TheHiveTLP {
  CLEAR = 0,
  GREEN = 1,
  AMBER = 2,
  "AMBER+STRICT" = 3, <--- I added the + symbol so the label can be used in the UI.
  RED = 4,
}
```

The reason is that by doing

```
Object.entries(TheHiveTLP).map(
  ([_, value], index) => ({
    text: value,
    value: index,
  })
);
```

we rely on the order of the enum which may change or be different in the future. If we explicitly map the key to an integer we avoid this problem. The key can be used as the value to be shown in the UI and there is no need to rely on the order of the enum. What do you think?

Reply:
Yes, That make more sense. Let me change it.